Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM node:12-slim
+COPY . ./
+RUN cd v2-website && npm i 
```

Comment:
Should use `website` rather than `v2-website`

Reply:
fixed in d4f30ed148a9269ac3b5606992d124f7b3f66d75